src/traces/contour/colorbar.js

```diff
@@ -30,11 +35,6 @@ module.exports = function colorbar(gd, cd) {
     var cb = drawColorbar(gd, cbId);
     cd[0].t.cb = cb;
 
-    var contours = trace.contours,
```

Comment:
Perhaps no good reason. I think this was path-dependent where I first had some extra conditions that required it, then removed the extra conditions. I'll work through this and attempt to keep the original flow intact.